tasks/testwasher.py

```diff
@@ -46,10 +46,13 @@ def merge_known_flakes(self, marked_flakes):
         """
         Merge flakes marked in the go code and the ones from the flakes.yaml file
         """
-        shared_packages = marked_flakes.keys() & self.known_flaky_tests.keys()
-        for package in shared_packages:
-            marked_flakes[package] |= self.known_flaky_tests[package]
-        return self.known_flaky_tests | marked_flakes
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** Isn't it clearer to use `intersection` and ‏`union` ?

Reply:
do you mean `intersection` instead of `&` and `union` instead of `|`?
As you wish (I'm a lazy boy so usually I use the shortest way :))
As this is the code I remove if I want to change what remains I cannot use the `|=` shortcut neither :/

Reply:
Yes I meant this, `a |= b` is ~ syntax sugar for `a = a | b` => `a = a.union(b)`

Reply:
As I said I'm lazy but I agree it's more readable :)